backend/src/ml_space_lambda/authorizer/lambda_function.py

```diff
@@ -283,8 +286,15 @@ def lambda_handler(event, context):
             ) and Permission.ADMIN in user.permissions:
                 policy_statement["Effect"] = "Allow"
             elif requested_resource == "/project" and request_method == "POST":
-                # Anyone can create a project
-                policy_statement["Effect"] = "Allow"
+                if Permission.ADMIN in user.permissions:
+                    policy_statement["Effect"] = "Allow"
+                else:
+                    # Get the latest app config
+                    app_config = _get_app_config()
+                    logger.info(f"Admin only: {app_config.configuration.project_creation.admin_only}")
```

Comment:
This log statement isn't especially clear, but non-blocking.

I think something to the effect of "Project creation restricted to Admins only: <>" more accurately reflects the status. Without looking at the line for this log statement, it would be hard to understand the context seeing this in a log file I think. Also a more verbose log here probably eliminates the need for the comment below.

Reply:
I think I just forgot to remove this after testing, thanks for spotting it